youtube_dl/extractor/abc.py

```diff
@@ -103,6 +103,65 @@ def _real_extract(self, url):
         }
 
 
+class ABCIViewShowIE(InfoExtractor):
+    IE_NAME = 'abc.net.au:iview:show'
+    _VALID_URL = r'https?://iview\.abc\.net\.au/(?:[^/]+/)*show/(?P<id>[^/?#]+)/?'
+    _GEO_COUNTRIES = ['AU']
+
+    # ABC iview programs are normally available for 14 days only.
+    _TESTS = [{
+        'url': 'https://iview.abc.net.au/show/sarah-and-duck/',
+        'info_dict': {
+            'id': 'sarah-and-duck',
+            'title': 'Sarah And Duck',
+            'description': 'Sarah is a 7 year old girl with big eyes and a green hat, who lives with her quacky, flappy, slightly manic, but endearing best friend, Duck.',
+        },
+        'playlist_count': 14,
+        'params': {
+            'skip_download': True,
+        },
+    }]
+
+    @classmethod
+    def suitable(cls, url):
+        return False if ABCIViewIE.suitable(url) else super(ABCIViewShowIE, cls).suitable(url)
+
+    def _real_extract(self, url):
+        show_id = self._match_id(url)
+        show_data = self._download_json('https://api.iview.abc.net.au/v2/show/' + show_id, show_id)
```

Comment:
Carry long lines.

Reply:
Put all the parts onto their own line? 

I'm not familiar with your preferred whitespace 'look', and this isn't enough info to guess.

Reply:
There are a number of locations where the lines have been split like this:

```
show_data = self._download_json(
    'https://api.iview.abc.net.au/v2/show/' + show_id, show_id)
```

Reply:
Cool, I'll make this one match.